### PR TITLE
add unique constraint to key column

### PR DESCRIPTION
**Pull request by mo7amed-3bdalla7**
'd key column be unique index to prevent conflict.

### CODE REVIEW

**Comment by anlutro** [comment]
Hey, thanks for the pull request, the change makes sense. I can't remember any reason for not making it unique.

I'm curious if this needs a new migration or if modifying the existing one is okay. I haven't been involved in Laravel development for years now so I don't know the answer to that question...

**Comment by bweston92** [comment]
It would be a migration to alter the schema.

**Comment by bweston92** [comment]
@mo7amed-3bdalla7 would you be able to add it to a new migration?

File: src/migrations/2015_08_25_172600_create_settings_table.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ public function up()
 		Schema::create($this->tablename, function(Blueprint $table)
 		{
 			$table->increments('id');
-			$table->string($this->keyColumn)->index();
+			$table->string($this->keyColumn);
```
Comment (bweston92) [inline src/migrations/2015_08_25_172600_create_settings_table.php]: Remove in the new migration, people are already using this migration.

**Comment by bweston92** [comment]
LGTM

**Pull request merged**